about.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="stylesheet" href="./css/about.css">
+    <title>About</title>
+</head>
+<body>
+    <header>
+        <nav class="nav1">
+            <ul>
+                <li>
+                    <i id="facebook-icon" class="fa fa-facebook"></i>
+                </li>
+                <li>
+                    <i id="twitter-icon" class="fa fa-twitter"></i>
+                </li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </nav>
+        <nav class="nav2">
+            <div class="openMenu"><i class="fa fa-bars"></i></div>
+            <ul class="mainMenu">
+                <a href="#" class="nav-logo">Tonny's Farm</a>
+                <li><a href="./index.html">Home</a></li>
+                <li><a href="./about.html">About</a></li>
+                <li><a href="">Program</a></li>
+                <li>Join</li>
+                <li>Sponsor</li>
+                <li>News</li>
+                <li id="border-line">Campaign</li>
+                <li class="closeMenu"><i class="fa fa-times"></i></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="landing">
+        <div class="about-landing-content">
```

Comment:
- [ ] The page is not responsive. Please make sure your website is totally responsive.
![image](https://user-images.githubusercontent.com/58504655/190701231-f8fa5596-b907-4628-842d-cbbe130dc575.png)